src/part2.md

```diff
@@ -0,0 +1,9 @@
+## Part 2
+
+What would be the time and space complexity of the algorithm used to format the data in part 1? 
+Please expand on why your approach to formatting the data is the best for this endpoint. 
+
+**Answer**:
+- The runtime complexity to format the data in part 1 would be O(n) where n is equal to the length of the transactions between the two given dates. 
```

Comment:
Yes, that is correct! Can you explain how you came to that conclusion?
Additionally, can you elaborate on what the space complexity if your solution is?